### DISCUSSION TITLE
Configure chrome PWA

### DISCUSSION THREAD

**Discussion by flzpenteado** [will-stone/browserosaurus#574] [Ideas]
Hello guys!
I really love browserossaurus and I'm wondering if there's a way to configure chrome PWAs on it (eg. Google Mail, Google Meet, etc).

**Comment by will-stone** [comment]
Browserosaurus uses the [app-exists](https://www.npmjs.com/package/app-exists) package to detect if apps are installed. If you can find a way for that to detect PWAs (or any other node.js method) then please let me know and/or submit a PR and we can see about supporting it in the project. Thanks 🙂
